updated content based on user suggestion (#4717)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/windows-defender-antivirus/configure-network-connections-windows-defender-antivirus.md

After you've enabled the service, you may need to configure your network or firewall to allow connections between it and your endpoints.

> As a cloud service, it is required that computers have access to the internet and that the ATP machine learning services are reachable. The URL: "*.blob.core.windows.net" should not be excluded from any kind of network inspection. The table below lists the services and their associated URLs. You should ensure there are no firewall or network filtering rules denying access to these URLs, or you may need to create an allow rule specifically for them (excluding the URL: "*.blob.core.windows.net).

Reviewer: The backslash escape character is needed to display actual punctuation characters instead of disappearing as MarkDown text format indicators.
```suggestion
As a cloud service, it is required that computers have access to the internet and that the ATP machine learning services are reachable. The URL: "\*.blob.core.windows.net" should not be excluded from any kind of network inspection. The table below lists the services and their associated URLs. You should ensure there are no firewall or network filtering rules denying access to these URLs, or you may need to create an allow rule specifically for them (excluding the URL: "\*.blob.core.windows.net).
```

> As a cloud service, it is required that computers have access to the internet and that the ATP machine learning services are reachable. The URL: "\*.blob.core.windows.net" should not be excluded from any kind of network inspection. The table below lists the services and their associated URLs. You should ensure there are no firewall or network filtering rules denying access to these URLs, or you may need to create an allow rule specifically for them (excluding the URL: "\*.blob.core.windows.net).

Reviewer 2: ```suggestion
As a cloud service, it is required that computers have access to the internet and that the ATP machine learning services are reachable. The URL: "\*.blob.core.windows.net" should not be excluded from any kind of network inspection. The table below lists the services and their associated URLs. You should ensure there are no firewall or network filtering rules denying access to these URLs, or you may need to create an allow rule specifically for them (excluding the URL: "\*.blob.core.windows.net").
```

> As a cloud service, it is required that computers have access to the internet and that the ATP machine learning services are reachable. The URL: "*.blob.core.windows.net" should not be excluded from any kind of network inspection. Below is the table lists the services and their associated URLs. You should ensure there are no firewall or network filtering rules that would deny access to these URLs, or you may need to create an allow rule specifically for them. (excluding URL: "*.blob.core.windows.net)

Reviewer: Syntax improvement suggestions.
```suggestion
As a cloud service, it is required that computers have access to the internet and that the ATP machine learning services are reachable. The URL: "*.blob.core.windows.net" should not be excluded from any kind of network inspection. The table below lists the services and their associated URLs. You should ensure there are no firewall or network filtering rules denying access to these URLs, or you may need to create an allow rule specifically for them (excluding URL: "*.blob.core.windows.net).
```
